openwhisk/tests/taskrun-install-npm-packages.yaml

```diff
@@ -0,0 +1,55 @@
+# Git Pipeline Resource representing an application source on GitHub
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: app-git
```

Comment:
You only need one PipelineResource to reference, not create two as you have already one in 
https://github.com/tektoncd/catalog/pull/375/files?file-filters%5B%5D=.jpg&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.sh&file-filters%5B%5D=.tmpl&file-filters%5B%5D=.yaml&file-filters%5B%5D=.zip#diff-1c0712a9233a5d7535a1712ebef1ea1aR1

Reply:
Originally, these "tests" were not meant for automation, but we happened to choose the same subdir. structure.  Removing them from the commit